.github/workflows/docs.yml

```diff
@@ -10,18 +10,13 @@ jobs:
     steps:
       - uses: actions/checkout@v2
         with:
-          fetch-depth: 1
+          persistent-credentials: false          
       - name: run asciidoctor
         uses: docker://asciidoctor/docker-asciidoctor
         with:
           args: asciidoctor index.adoc -o index.html
```

Comment:
we should probably change this to `asciidoctor *.adoc`, since only index.html is being generated at the moment